src/main/java/ch/njol/skript/sections/EffSecCreateWorldBorder.java

```diff
@@ -0,0 +1,136 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.sections;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.EffectSection;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.WorldBorder;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("World Border Creation")
+@Description({
+	"Create a world border. This section is used to create an empty world border",
+	"You can modify the world border in this section, using for example 'the world border'."
+})
+@Examples({
+	"create a world border:",
+	"\tset center of border to player's location",
+	"set player's world border to the last created world border"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("1.18+")
+public class EffSecCreateWorldBorder extends EffectSection {
```

Comment:
As discussed, I think we should decided whether we want to support syntax like `the last worldborder`

I personally think just having SecCreateWorldBorder is enough. If storing the worldborder is desired, it could be done within the section. I also understand that some may prefer a regular Section + an expression (ex: `set {_test} to a new world border`), so I think discussion of that is okay here too